test/app.spec.js

```diff
@@ -743,7 +741,7 @@ describe('app', () => {
 
     beforeEach(() => {
       messenger = new Messenger({ cache: new Cacheman('test') });
-      sandbox.stub(messenger, 'getPublicProfile').resolves({});
+      sandbox.stub(messenger, 'getPublicProfile').resolves({ first_name: 'Gregor' });
```

Comment:
_Game of Thrones_, _Ronin_, or some dude from X Factor?

Reply:
my test name is "Gregor Samsa"